.github/actions/keras_application_test/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Keras2onnx Application Test Run
```

Comment:
Do you need an extra folder?

Reply:
Such path is required by Github Workflow, so we don't need to customize them more.